src/gam.py

```diff
@@ -7867,7 +7867,8 @@ def doShowServiceAccountKeys():
     print('{0}: {1}'.format(parts[i][:-1], parts[i+1]))
   for key in keys:
     key['name'] = key['name'].rsplit('/', 1)[-1]
-    key['current'] = key['name'] == currentPrivateKeyId
+    if key.get('keyType', '') == 'USER_MANAGED':
```

Comment:
I'd prefer to just set current=true on the one key and leave it unset instead of false for everything else. Also current is a bit ambiguous, we should make it a bit more clear that it's the key GAM is currently using. Maybe, used_to_authenticate_this_request ?

Additionally, I'm not opposed to a command that will revoke a single key along the lines of:

gam delete sakey <id>

what I don't want is commands like "gam create sakey" that will make it to easy to end up with a whole bunch of keys that are a challenge for an admin to identify and manage, that just weakens our security story while adding no real value (admins shouldn't need to do that).